src/github-handler/comment-handler/make-review-handler/upload-comments-handler.ts

```diff
@@ -99,15 +97,19 @@ export async function makeInlineSuggestions(
   ).data.head.sha;
   const comments = buildReviewComments(suggestions);
   if (!comments.length) {
-    logger.info('No suggestions to make. Exiting...');
-    return;
+    logger.info('No suggestions to make');
+  }
+  const errorMessage = buildErrorMessage(outOfScopeSuggestions);
+  if (errorMessage) {
+    logger.warn('Some suggestions could not be made');
```

Comment:
I think we're missing the case of no suggestion and no errors (which should do nothing).

Reply:
Hm wouldn't it be the case: when we have no suggestions we would still want to comment that out of scope suggestions were generated?

Reply:
I propose we should do nothing if we have both no suggestions to make and no out of scope suggestions generated (assuming users care about all suggestions generated). Do you agree?

Reply:
Correct, if there are no suggestions and no errors, we should do nothing.